src/vs/workbench/contrib/scm/browser/repositoryPane.ts

```diff
@@ -796,6 +785,12 @@ export class RepositoryPane extends ViewPane {
 		this._register(this.inputEditor.onDidFocusEditorText(() => addClass(editorContainer, 'synthetic-focus')));
 		this._register(this.inputEditor.onDidBlurEditorText(() => removeClass(editorContainer, 'synthetic-focus')));
 
+		const updateInputEditorOptions = () => {
+			const newEditorOptions = this.getInputEditorOptions();
+			this.inputEditor.updateOptions(newEditorOptions);
```

Comment:
We don't really need to extract `getInputEditorOptions` into a method, since the `editor.updateOptions` will work fine if you just pass the option you want changed. There's no need to pass it all options once again.